apps/laboratory/app/layout.tsx

```diff
@@ -11,6 +11,13 @@ export default async function App({ children }: { children: React.ReactNode }) {
 
   return (
     <html>
+      <title>AppKit Lab</title>
```

Comment:
Can we instead use Metadata object? https://nextjs.org/docs/app/building-your-application/optimizing/metadata#static-metadata